### ISSUE TITLE
CleanUpConfigs

### ISSUE THREAD

**Issue by Tharki-God** [TheCommieAxolotl/BetterDiscord-Stuff#6]
### Addon

CleanUpConfigs

### What happened?

Error reading Plugins Folder' TypeError: Cannot read property 'Sizes' of undefined
![image](https://user-images.githubusercontent.com/73281112/187563999-d5365c92-54f7-4efc-a67c-09bbd0450052.png)

### What Discord version do you have this issue on?

Stable

### Logs

```shell
[CleanUpConfigs] Error reading Plugins Folder' TypeError: Cannot read property 'Sizes' of undefined
(anonymous)	@	014ecc8….js:formatted:8337
t.<computed>	@	300d3f1….js:formatted:187491
overrideMethod	@	react_devtools_backend.js:4026
(anonymous)	@	VM3681:4
_log	@	0PluginLibrary.plugin.js:1497
err	@	0PluginLibrary.plugin.js:1453
error	@	0PluginLibrary.plugin.js:3449
eval	@	CleanUpConfigs.plugin.js:328
(anonymous)	@	VM3237 fs.js:171
```

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by Tharki-God** [comment]
found the fix use `BdApi.findModuleByDisplayName("LegacyText")` instead of `BdApi.findModuleByDisplayName("Text")`

**Comment by Tharki-God** [comment]
and btw add message logger data files to the exclusion list which are MessageLoggerV2Data.config.json and MessageLoggerV2DataBackup.config.json

**Comment by TheCommieAxolotl** [comment]
Fixed with [765d040](https://github.com/TheCommieAxolotl/BetterDiscord-Stuff/commit/765d04044afe15cc49e50bc6176a4cb60f6ae236)
